src/Symfony/Component/HttpKernel/Config/FileLocator.php

```diff
@@ -46,6 +49,10 @@ public function __construct(KernelInterface $kernel, string $path = null, array
     public function locate($file, $currentPath = null, $first = true)
     {
         if (isset($file[0]) && '@' === $file[0]) {
+            if (null !== $this->path) {
+                @trigger_error('Using the global fallback to load resources is deprecated since Symfony 4.4 and will be removed in Symfony 5.0.', E_USER_DEPRECATED);
```

Comment:
This will always trigger a deprecation. What I had in mind was only triggering a deprecation when a resource is actually loaded from the global dir.

Reply:
done